scripts/install_ddev.sh

```diff
@@ -98,12 +98,19 @@ VERSION=$LATEST_VERSION
 if [ $# -ge 1 ]; then
   VERSION=$1
 fi
-RELEASE_BASE_URL="https://github.com/${GITHUB_USERNAME}/ddev/releases/download/$VERSION"
 
-rv=$(semver_compare "${VERSION}" "v1.10.0")
-if [[ ${rv} -lt 0 ]]; then
-  printf "${RED}Sorry, this installer does not support specifying versions of ddev prior to v1.10.0${RESET}\n"
-  exit 1
+if [ "$VERSION" = 'HEAD' ]; then
+  RELEASE_BASE_URL="https://nightly.link/drud/ddev/workflows/master-build/master"
+else
+  RELEASE_BASE_URL="https://github.com/${GITHUB_USERNAME}/ddev/releases/download/$VERSION"
+fi
+
+if [ "$VERSION" != 'HEAD' ]; then
+  rv=$(semver_compare "${VERSION}" "v1.10.0")
+  if [[ ${rv} -lt 0 ]]; then
+    printf "${RED}Sorry, this installer does not support specifying versions of ddev prior to v1.10.0${RESET}\n"
+    exit 1
```

Comment:
Why limit?

Reply:
@rfay?

Reply:
I can't remember, but I think the artifacts were in a different naming format before v1.10.